.github/workflows/buildAndPush.yaml

```diff
@@ -33,7 +38,7 @@ jobs:
           password: ${{ secrets.QUAYIO_OCPMETAL_PASSWORD }}
           registry: quay.io
           dockerfile: Dockerfile
-          tags: 'latest,${{ github.sha }}'
+          tags: 'latest,${{ github.sha }},${{ env.GIT_TAG }}'
```

Comment:
we should not use this here but in "on tag" only. Makes probably no sense here, just confusing.

Reply:
done